tests/TestCase/Model/Behavior/DuplicatableBehaviorTest.php

```diff
@@ -180,4 +181,25 @@ public function testWithTranslation()
             ->all();
         $this->assertEquals(2, $records->count());
     }
+
+    public function testDuplicateWithSetters()
```

Comment:
This new test cases passes even without the change you have done to the behavior, so it's not really useful :slightly_smiling_face:.

In general when fixing a bug writing a failing test first and then fix the code.

Reply:
🤔 I'll have to revise this test.